Bottom: a78169542eca04ec34a3816917462e0dfb6a1eb6
Top:    0f2165b2dfdbc6f63ebc5485c118b7b9bf464ac9
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-21 16:48:05 -0700

Refresh of update-gitignore

---

diff --git a/01_12/Makefile b/01_12/Makefile
new file mode 100644
index 0000000..fd9f01a
--- /dev/null
+++ b/01_12/Makefile
@@ -0,0 +1,5 @@
+main:
+	g++ -o main main.cc
+
+clean:
+	rm main
